transforms/CMakeLists.txt

```diff
@@ -6,10 +6,12 @@ find_package(LLVM REQUIRED CONFIG)
 list(APPEND CMAKE_MODULE_PATH "${LLVM_DIR}")
 list(APPEND CMAKE_MODULE_PATH "${LLVM_CMAKE_DIR}")
 
-# needed for LLVM 3.7
-set(LLVM_RUNTIME_OUTPUT_INTDIR ${CMAKE_BINARY_DIR}/${CMAKE_CFG_INTDIR})
-set(LLVM_LIBRARY_OUTPUT_INTDIR ${CMAKE_BINARY_DIR}/${CMAKE_CFG_INTDIR})
-include(HandleLLVMOptions)
+if (${LLVM_PACKAGE_VERSION} VERSION_LESS_EQUAL "3.7")
+  # needed for LLVM 3.7
+  set(LLVM_RUNTIME_OUTPUT_INTDIR ${CMAKE_BINARY_DIR}/${CMAKE_CFG_INTDIR})
+  set(LLVM_LIBRARY_OUTPUT_INTDIR ${CMAKE_BINARY_DIR}/${CMAKE_CFG_INTDIR})
+  include(HandleLLVMOptions)
+endif()
```

Comment:
Is this really needed for LLVM 3.7? I was considering removal, because the [documentation for 3.7](https://releases.llvm.org/3.7.0/docs/CMake.html#developing-llvm-passes-out-of-source) looks exactly the same as for LLVM 10 in this matter, it does not mention inclusion `HandleLLVMOptions` at all.

Reply:
Hmm, I guess if it would not be needed then we would not add it in cfbc4afad8cb. There may have been a bug that may be already fixed in the newest minor version of LLVM 3.7. To be honest, I do not know if it is needed or not, but it probably was two years ago.

Reply:
I have just tested `transforms` with LLVM 3.7.{0,1} built from sources and it configures and builds just fine with some minor changes to the code itself. So, I assume its safe to remove this completely. 

I can send the patches for LLVM 3.7 support in a separate PR if you'd like to have them included too.

Reply:
Ok, let's go with this removed and if there appears a problem, we can always revert this part.